src/components/common/header/Notification/Notification.tsx

```
@@ -1,13 +1,18 @@
+import useAlarmList from '../../../../hooks/useAlarmList';
+import LoadingSpinner from '../../loadingSpinner/LoadingSpinner';
 
 const Notification = () => {
   const { alarmListData: AlarmData, isLoading, isFetching } = useAlarmList();
 
   if (!AlarmData) {
-    return <S.Message>알림이 없습니다.</S.Message>;
+    return (
+      <S.Container>
+        <S.Message>알림이 없습니다.</S.Message>;
+      </S.Container>
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)